src/index.html

```diff
@@ -6,7 +6,39 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="body">
+    <main class="calendar calendar--mounth--mounth29">
+      <div class="day day--day--sunday"></div>
```

Comment:
start-day modifier should be also on the parent block, not on the first child element